consts/index.ts

```
@@ -0,0 +1,134 @@
+export const skillCategories = [
+  {
+    category: "Languages",
+    skills: ["JavaScript", "TypeScript", "C++", "GoLang", "Python"],
+  },
+  {
+    category: "Frontend",
+    skills: ["React.js", "Next.js", "Tailwind CSS", "ShadCN"],
+  },
+  {
+    category: "Backend",
+    skills: ["Node.js", , "Express.js"],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)